js/constants.js

```diff
@@ -261,3 +261,10 @@ export const JOIN_POST_URL = `${BASE_URL}/users/self/intro`;
 export const SELF_URL = `${BASE_URL}/users/self`;
 export const GITHUB_OAUTH =
```

Comment:
can we change it to `api.realdevsquad.com/auth/github/login ` with respect to the following commit: https://github.com/Real-Dev-Squad/website-backend/commit/c8f53da7970aea9685a50d7c2a622625b4aee881